fhirest-core/src/main/java/ee/fhir/fhirest/core/service/conformance/HapiContextHolder.java

```diff
@@ -121,7 +128,13 @@ protected IValidationSupport getValidationSupport() {
         .filter(c -> SUPPORTED_CS_CONTENT_MODES.contains(c.getContent()))
         .collect(Collectors.toMap(d -> d.getUrl(), d -> d, (a, b) -> a));
 
+    CacheConfiguration cacheConfiguration = defaultValues();
+    cacheConfiguration.setCacheTimeout(
+            Duration.ofMinutes(minuteCacheTimeout)
+    );
+
     ValidationSupportChain chain = new ValidationSupportChain(
+            cacheConfiguration,
```

Comment:
just to keep with the style in this class, might consider inline `CacheConfiguration.defaultValues().setCacheTimeout(cacheTimeout)`